src/app/components/challenge/challengeoverview/challengeoverview.component.ts

```diff
@@ -75,7 +76,9 @@ export class ChallengeoverviewComponent implements OnInit {
 
     const PARAMS = {
       title: 'Edit Challenge Description',
-      content: 'asdasdasd \n\nasdasdasdasd',
+      content: '',
```

Comment:
The api needs? this `content` param? If not you don't need to send it in PARAMS.
```suggestion
```

Reply:
yeah we can remove that, done